<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sim_U4_T6</class>
 <widget class="QWidget" name="sim_U4_T6">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sim_U4_T6</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#sim_U4_T6,
QScrollArea#scrollArea{
background-color:rgba(0,0,0,0);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>548</width>
        <height>246</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#scrollAreaWidgetContents{
background-color:rgb(0,0,0);
}
QLabel{
color:white;
}
QTableWidget::item {
background-color: black;
color: white;
border: 1px solid rgb(80, 80, 80); 
}
QTableWidget::item:selected {
background-color: rgb(0,51,112);
}
QTableWidget QWidget,
QTableWidget QFrame{
background-color: rgb(173,51,51);
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Operaciones Morfol√≥gicas</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CustomPlotWidget" name="cp_1" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
         <item>
          <widget class="CustomPlotWidget" name="cp_2" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,0,0,1">
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QComboBox" name="cb_oper">
             <item>
              <property name="text">
               <string>Gradiente dy</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Gradiente dx</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Roberts 1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Roberts 2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Segunda Derivada</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Laplaciana</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Prewitt Horizontal</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Prewitt Vertical</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Diagonal 1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Diagonal 2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Sobel 1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Sobel 2</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Matriz de Operador</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,1">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>2</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="CustomTable" name="ct_oper" native="true">
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>10</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>2</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomPlotWidget</class>
   <extends>QWidget</extends>
   <header>clases.PlotWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CustomTable</class>
   <extends>QWidget</extends>
   <header>clases.CustomTable</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
